test/controllers/RecordsControllerSpec.scala

```diff
@@ -0,0 +1,22 @@
+package controllers
```

Comment:
Should add a test to make sure an unauthorised user is redirected to the login page as well. The SeriesDetailsControllerSpec test has an example of this.